src/storage/src/hummock/mod.rs

```diff
@@ -295,8 +269,8 @@ impl HummockStorage {
     }
 
     #[cfg(any(test, feature = "test"))]
```

Comment:
conflict with https://github.com/risingwavelabs/risingwave/pull/5934?

Reply:
#5934 has been reverted in #5937 with a better fix.